File: shared/src/main/java/com/emanuelvini/nuven/core/shared/data/repository/PlayerPreferences.java
package com.emanuelvini.nuven.core.shared.data.repository;

import com.emanuelvini.nuven.core.shared.player.preferences.lobby.visibility.VisibilityValue;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class PlayerPreferences {

    private String name;

    private VisibilityValue visibilityValue;

}

File: bungee/src/main/kotlin/com/emanuelvini/nuven/core/bungee/repository/PlayerPreferences.kt
package com.emanuelvini.nuven.core.bungee.repository

import com.emanuelvini.nuven.core.shared.player.preferences.Preference
import com.emanuelvini.nuven.core.shared.player.preferences.lobby.visibility.VisibilityValue

class PlayerPreferences(
    val name: String,
    val values : HashMap<Preference, Int>
)